content/SSH/Securing_SSH_with_FIDO2.adoc

```diff
@@ -1,173 +1,254 @@
+----
+Replace `Description` with a unique identifier for each credential.
 
-. SSH will save two files. A user can check their console and to display the actual filenames. By default, these filenames will be `id_ecdsa_sk` & `id_ecdsa_sk.pub`, but may be different dependent on whether or not it was changed to something else when prompted for a save location. In the example below, the default names are used.
-+
---
-The first file, `id_ecdsa_sk`, contains a key handle used to derive the private key credential on the YubiKey.  The second file ,`id_ecdsa_sk.pub`, contains the public key which is used on a remote system to verify authentication.
---
+== SSH to remote host using FIDO2
+This example will ssh to GitHub. link:https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account[It assumes the ssh public key has already been added to the GitHub account].
 
-. The public key can be added to a remote server with the following command:
+. . Open a terminal and use:
 +
---
-`ssh-copy-id -i ~/.ssh/id_ecdsa_sk.pub user@host`
---
-Note that on Windows, `ssh-copy-id` is not available, so the public key must be added to the remote host's `.ssh/authorized_keys` file by some other means.
-
-==== Using non-discoverable keys on another local system configuration
-. Follow the previous instructions and insert the configured YubiKey into system.  
+[source,sh]
+----
+ssh -T git@github.com
+----
++
+. Enter the PIN and touch the key when prompted.
 
-. Copy the `id_ecdsa_sk` file and `id_ecdsa_sk.pub` from step 4 in the previous instructions to the` ~/.ssh` folder on the new local system.
+== Troubleshooting
+*Password Prompts Instead of YubiKey:*
 
-. SSH to the remote system and touch the key when prompted.
+* Restart or log out/in.
+* Verify OpenSSH version (ssh -V).
+* Check system logs for errors:
+** Ubuntu/Debian:
++
+[source,sh]
+----
+tail /var/log/syslog | grep sshd
+----
++
+** Fedora:
++
+[source,sh]
+----
+journalctl -r /usr/sbin/sshd
+----
++
+* Run SSH in debug mode:
++
+[source,sh]
+----
+ssh -vvvv user@host
+----
 
-=== Using FIDO2 Keys with Windows Subsystem for Linux (WSL) on Windows
-In addition to a native SSH client, the Windows OpenSSH beta release also contains an `SSH_SK_HELPER` that can be used to bridge the host's FIDO2 support to WSL.  All of this configuration must be done from inside the WSL environment, and relies on the Windows environment to be working correctly.
+*Permission Issues:*
 
-. The `SSH_SK_HELPER` environment variable must be set to the path of the `ssh-sk-helper.exe` binary from the OpenSSH beta release.  For example:
+* Ensure correct file permissions:
 +
--- 
-`export SSH_SK_HELPER="/mnt/c/Program Files/OpenSSH/ssh-sk-helper.exe"`
---
+[source,sh]
+----
+chmod 600 ~/.ssh/id_ed25519_sk
+----
 
-. The SSH identity files (id_ecdsa_sk and  id_ecdsa_sk.pub) must be available in `~/.ssh/`
+*Verify the FIDO2 credential on security key*
```

Comment:
This is more useful when compared to the credential id in the private key file, but that requires additional tooling I think?

Reply:
Is the recommendation to remove this section?